data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,143 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+
+# ahca_polls ----------------------------------------------------------------------
+ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv")
+# snake_case for variable names
+colnames(ahca_polls) <- colnames(ahca_polls) %>%
+  tolower()
+# Change start and end dates to proper format
+ahca_polls$start <- as.Date(ahca_polls$start, "%m/%d/%y") 
+ahca_polls$end <- as.Date(ahca_polls$end, "%m/%d/%y")
+# Convert to factor variable
+ahca_polls$pollster <- as.factor(ahca_polls$pollster)
+devtools::use_data(ahca_polls, overwrite = TRUE)
+
+
+# bachelorette ---------------------------------------------------------------------
+bachelorette <- read_csv("data-raw/bachelorette/bachelorette.csv")
+# snake_case for variable names
+colnames(bachelorette) <- colnames(bachelorette) %>%
+  tolower() %>%
+  str_replace_all("-", "_")
+# Remove extra headers within data
+bachelorette<-bachelorette[!(bachelorette$season=="SEASON"),]
+# Convert character variables to factors
+cols.num <- c("season", "elimination_1","elimination_2", "elimination_3", "elimination_4", 
+              "elimination_5", "elimination_6", "elimination_7", "elimination_8", 
+              "elimination_9", "elimination_10", "dates_1", "dates_2", "dates_3", "dates_4", 
+              "dates_5", "dates_6", "dates_7", "dates_8", "dates_9", "dates_10")
+bachelorette[cols.num] <- lapply(bachelorette[cols.num], as.factor)
+devtools::use_data(bachelorette, overwrite = TRUE)
+
+
+# candy-power-rankings -------------------------------------------------------------
+candy_rankings <- read_csv("data-raw/candy-power-ranking/candy-data.csv")
+# snake_case for variable names
+candy_rankings <- rename(candy_rankings, c("competitorname"="competitor_name", 
```

Comment:
I'm getting an error here. I think it's because you are using `plyr::rename()` and not `dplyr::rename()`. The great thing about dplyr is the functions have obvious names; the bad thing about dplyr is the functions have obvious names and often conflict with other package. Could you change this to work with `dplyr::rename`?